tests/__mocks__/@myparcel/js-sdk/dist/data/delivery-options/getDeliveryOptionsEntry.js

```diff
@@ -37,47 +79,32 @@ export function getDeliveryOptionsEntry(today, isExtraDropOffDay) {
               },
             },
           ],
-          shipment_options: [
-            {
-              name: 'large_format',
-              schema: {
-                type: 'boolean',
-                enum: [
-                  true,
-                  false,
-                ],
-              },
-            },
-            {
-              name: 'only_recipient',
-              schema: {
-                type: 'boolean',
-                enum: [
-                  true,
-                ],
-              },
-            },
+          shipment_options: morningEveningShipmentOptions,
+        }],
+      ...isExtraDropOffDay
+        ? []
+        : [{
+          type: 'evening',
+          package_type: 'package',
+          delivery_time_frames: [
             {
-              name: 'signature',
-              schema: {
-                type: 'boolean',
-                enum: [
-                  true,
-                  false,
-                ],
+              type: 'start',
+              date_time: {
+                date: `${formattedDate} 08:00:00.000000`,
+                timezone_type: 3,
+                timezone: 'Europe/Amsterdam',
               },
             },
             {
-              name: 'return',
-              schema: {
-                type: 'boolean',
-                enum: [
-                  true,
-                  false,
-                ],
+              type: 'end',
+              date_time: {
+                date: `${formattedDate} 12:00:00.000000`,
+                timezone_type: 3,
```

Comment:
Mochten we ooit de timezone_type belangrijk vinden, om deze uit te schrijven naar constanten. ik weet dat `timezone_type: 1` te maken heeft met UTC, maar weet niet echt het verschil met 3 qua output.

Reply:
Dit hele bestand is een api response